packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/ZoomInButton/ZoomInButtonContainer.js

```diff
@@ -43,7 +43,7 @@ function ZoomInButtonContainer({ separateFrameZoomIn = () => true }) {
   return (
     <ZoomInButton
       disabled={disabled}
-      onClick={separateFramesView ? separateFrameZoomIn : zoomIn}
+      onClick={zoomCallback}
```

Comment:
This fires continuously while the Enter key is held down, but I can't see why.

Reply:
Test pen here: https://codepen.io/eatyourgreens/pen/abQaNqp

This seems to be the way that buttons work by default, at least in MacOS.